src/mixins/date-mixin.ts

```diff
@@ -28,9 +28,12 @@ export default class DateMixin extends Vue {
    * @returns the formatted date string
    */
   toReadableDate (date: string): string {
-    // cast to a workable dateString
+    // cast to a workable dateString 'America/Vancouver'
```

Comment:
No, the tz depends on the locale for this code (ie, browser). This comment could mislead.

Reply:
Sorry, that an error. I was testing some things out and missed removing that comment